src/Jackett.Common/Definitions/ygg-api.yml

```
@@ -0,0 +1,184 @@
+---
+id: yggapi
+name: Ygg API
+description: Indexeur non officiel pour ygg.re
+language: fr-FR
+type: private
+encoding: UTF-8
+followredirect: true
+testlinktorrent: false
+links:
+  - https://yggapi.eu
```

```suggestion
  - https://yggapi.eu/
```
standard for links is to end with a slash. don't forget to remove the slash at the start of all the search paths ;-)

---

Done ;)